core/src/main/kotlin/org/evomaster/core/problem/graphql/Table.kt

```diff
@@ -1,9 +1,15 @@
 package org.evomaster.core.problem.graphql
+
+import org.evomaster.core.problem.graphql.schema.__TypeKind
+
 /**
- * Intermediate structure to parse and organize the object graphQl-schema types into three components: Name, Field, Type
+ * Intermediate structure to parse and organize the object graphQl-schema types into five components: Name, Field,
+ * kindOfTableField, Type, kindOfTableType
  * */
 class Table {
     var tableField : String?=null
-    var tableType : String?=null
+    var tableType : String= ""
+    var kindOfTableField: __TypeKind?=null
     var tableName : String?=null
+    var kindOfTableType: __TypeKind?=null
 }
```

Comment:
are these fields supposed to change once an instance of `Table` is created? if not, then can use `val`